src/more_ops.rs

```diff
@@ -720,7 +720,7 @@ fn test_op_lsh() {
     );
 
     let node = test_shift(op_lsh, &mut a, &[1], &[0x80, 0]).unwrap().1;
-    assert_eq!(a.atom(node).as_ref(), &[]);
+    assert_eq!(a.atom(node).as_ref().len(), 0);
```

Comment:
Just curious, why is this necessary?

Reply:
the compiler found an ambiguous interpretation of the type of this empty slice. I didn't dig into understanding exactly why, but I believe some other trait may have been pulled into the context.
It was quite mysterious to me too. Perhaps these only showed up together with some other errors, and could be reverted.
The error message was talking about some JSON type that also matched here. I don't recall off-hand